### ISSUE TITLE
item filter enforcing only order without proximity

### ISSUE THREAD

**Issue by ajoslin103** [TablePlus/TablePlus#1979]
I am noticing that the item filter, when given a simple string fragment, is returning all matches that contain all the letters in the fragment in order but without proximity

I entered "acc" (without the quotes) and received this file listing:
```
wpjt_app_cell_carriers
wpjt_app_certifications
wpjt_app_contact_options
wpjt_app_contact_materials
wpjt_app_curriculum_materials
```

I expected to find: no results

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
MySQL 5.6.550-84.0 (on a SiteGround server)

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Build 322

3. The steps to reproduce this issue:
Open connection to the database, try to filter items, notice that all chars are required without proximity

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
HI @ajoslin103 we use fuzzy search in all search field. You can force TablePlus to use contains by adding the double quote.

For example: "abc" instead of abc

**Comment by ajoslin103** [comment]
Ah, I see -- one must close the double quote to activate contains, and so one can embed within a fuzzy search

Nice!  And in experimenting I see we can use |'s as well.

It would be great if there was a 'searching cheatsheet' to educate users on how to use the fuzzy search.  It would be fantastic if it's location was "advertised" via the placeholder.  And it would be amazing if access to the 'searching cheatsheet' was via an icon -- the "clear search field" icon disappears, leaving a usable space when there is nothing in the searching field...

You have created a fantastic tool here!!  

I would like very much to understand how to use the Mongo (beta) tool 

But even more if you could add GraphQL capabilities similar to the Insomnia tool 

I am grateful to have my questions answered by the Founder 

Please keep yourself safe and well

**Comment by huyphams** [comment]
> It would be great if there was a 'searching cheatsheet' to educate users on how to use the fuzzy search. It would be fantastic if it's location was "advertised" via the placeholder. And it would be amazing if access to the 'searching cheatsheet' was via an icon -- the "clear search field" icon disappears, leaving a usable space when there is nothing in the searching field...

Yeah, it's really nice idea. A (?) button placed at the end of the search field could help and tooltip!
